src/editor/reducers/notebook-reducer.js

```diff
@@ -329,6 +329,38 @@ const notebookReducer = (state = newNotebook(), action) => {
       });
     }
 
+    case "MOVE_TO_NEXT_CHUNK": {
```

Comment:
you shouldn't need to touch the reducer to get this done -- take a look at the actions
`updateEditorCursor`and `moveCursorToNextChunk`